.github/workflows/ci.yml

```
@@ -12,6 +12,13 @@ env:
   NODE_OPTIONS: --max-old-space-size=6144
 
 jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

suggestion: add a comment explaining why this setting is used
```suggestion
          # required by wagoid/commitlint-github-action
          fetch-depth: 0
```

---

@LayZeeDK Done, thanks!